Program.cs

```diff
@@ -236,7 +236,7 @@ public ValidationResult DotnetResult
     
     private CancellationTokenSource _validatorCancellationSource;
     
-    private List<Process> _validatorProcesses = new List<Process>();
+    private readonly List<Process> _validatorProcesses = new List<Process>();

```

Comment:
This confused me a bit, but yeah, this is totally correct :)

Reply:
I had to think about it too!